src/api/grainConfig.js

```diff
@@ -11,6 +11,7 @@ export type GrainConfig = {|
   +balancedPerWeek?: number,
   +recentPerWeek?: number,
   +recentWeeklyDecayRate?: number,
+  +numPeriodsLookback?: number,
```

Comment:
if we add it in the config, it should probably say `immediateNumPeriodsLookback` so that it's clear which policy it applies to.

That said, we should really come up with a backwards compatible way to add arbitrary policies, rather than continuing to add extra fields here. Could we do that instead of adding another field? I'm thinking there would be a field like `policies` which is an optional array of explicit allocation policies, and the hardcoded per-policy fields just implicitly add extra policies. But then we could migrate everyone to just use the policies field. (This would be a separate PR.)

Reply:
I agree.  I can open #2513 back up.  Should have made a clearer case/vision when I opened that one.  That also lets us take multiple of the same policy type, which is nice.

Reply:
Done.